<launch>

  <node name="image_to_pcl_node" pkg="convert_pcl" type="image_to_pcl_node" output="screen">

    <param name="scale"                 value="2" />
    <param name="intrinsic_filename"    value="$(find convert_pcl)/intrinsics.yml" />
    <param name="extrinsic_filename"    value="$(find convert_pcl)/extrinsics.yml" />
    <param name="disparity_size"        value="128" />
  </node>

  
</launch>
